src/GitHub.App/ViewModels/StartPageCloneViewModel.cs

```diff
@@ -0,0 +1,208 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.Composition;
+using System.Diagnostics;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Reactive;
+using System.Reactive.Linq;
+using System.Text.RegularExpressions;
+using System.Windows.Input;
+using GitHub.App;
+using GitHub.Exports;
+using GitHub.Extensions;
+using GitHub.Models;
+using GitHub.Services;
+using GitHub.Validation;
+using NLog;
+using NullGuard;
+using ReactiveUI;
+using Rothko;
+using System.Collections.ObjectModel;
+using GitHub.Collections;
+using GitHub.UI;
+using GitHub.Extensions.Reactive;
+
+namespace GitHub.ViewModels
+{
+    [ExportViewModel(ViewType=UIViewType.StartPageClone)]
+    [PartCreationPolicy(CreationPolicy.NonShared)]
+    public class StartPageCloneViewModel : BaseViewModel, IBaseCloneViewModel
```

Comment:
I don't plan on keeping this specific viewmodel around for long but right now there's Reasons(tm) for it.